### PR TITLE
Add debug logs

### PR DESCRIPTION
**Pull request by sam-nguyen7**
I'm primarily concerned with 
1. What the initial list of ingresscontroller looks like on cluster
2. What we have in the publshingstrategy
3. The list of the difference. aka what we have in publishingstrategy that is NOT on cluster. - these are the ones that the reconcile should action on.

### CODE REVIEW

File: pkg/controller/publishingstrategy/publishingstrategy_controller.go [hunk 1]
```diff
@@ -139,6 +139,12 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 	}
 
+	// ----------------------------TODO: remove these debug logs afterwards --------------------------------------
+	log.Info(fmt.Sprintf("initial ingresscontroller list: %v", ingressControllerList.Items))
+	log.Info(fmt.Sprintf("appingress on publishingstrategy CR: %v", instance.Spec.ApplicationIngress))
+	log.Info(fmt.Sprintf("ingress to reconcile: %v", ingressNotOnCluster))
```
Comment (lisa) [inline pkg/controller/publishingstrategy/publishingstrategy_controller.go:145]:
```suggestion
	log.Info(fmt.Sprintf("initial ingresscontroller list: %+v", ingressControllerList.Items))
	log.Info(fmt.Sprintf("appingress on publishingstrategy CR: %+v", instance.Spec.ApplicationIngress))
	log.Info(fmt.Sprintf("ingress to reconcile: %+v", ingressNotOnCluster))
```

`%+v` will probably be a bit better to print field names.

File: pkg/controller/publishingstrategy/publishingstrategy_controller.go [hunk 2]
```diff
@@ -139,6 +139,12 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 	}
 
+	// ----------------------------TODO: remove these debug logs afterwards --------------------------------------
```
Comment (2uasimojo) [inline pkg/controller/publishingstrategy/publishingstrategy_controller.go:142]: If you wanted to keep these messages long-term, you could use log.V(some_high-ish_number).Info(...). That way you could switch them on in a given cluster without a code change in the future.

**Pull request merged**